platforms/hyperledger-indy/configuration/deploy-network.yaml

```diff
@@ -70,6 +70,23 @@
       loop_var: organizationItem
     when: network['type'] == 'indy'
 
+  # Admin K8S auth
+  - name: Admin K8S auth
+    include_role:
+      name: setup/vault_kubernetes
+    vars:
+      organization: "{{ organizationItem.name | lower }}"
+      component_ns: "{{ organizationItem.name | lower }}-ns"
+      component_name: "{{ organization }}-baf-ac-vault-auth"
+      component_type: "GetServiceAccount"
+      vault: "{{ organizationItem.vault }}"
+      auth_path: "{{ organization }}-net-auth"
```

Comment:
The auth_path doesn't match

https://github.com/hyperledger-labs/blockchain-automation-framework/blob/9cad77b8bad2a6ee461b3bde125d6dc4d0459e1b/platforms/hyperledger-indy/charts/indy-key-mgmt/templates/job.yaml#L34

I would use e.g. kubernetes-{{ organization }}-admin-auth in both places

Reply:
updated at both places